src/Arcus.Templates.AzureFunctions.Http/.template.config/ide.host.json

```
@@ -15,7 +15,14 @@
       "name": {
         "text": "Include Health Checks"
       },
-      "isVisible": true 
+      "isVisible": true
+    },
+    {
+      "id": "exclude-openapi",
+      "name": {
+        "text": "Exclude OpenApi"
```

```suggestion
        "text": "Exclude OpenApi Documentation"
```
That's maybe a bit more descriptive ?

---

Yeah, that would be great 😄 .